src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1257,7 +1247,9 @@ static void uct_ib_mlx5_devx_cleanup_flush_mr(uct_ib_mlx5_md_t *md)
         return;
     }
 
-    uct_ib_mlx5_devx_obj_destroy(md->flush_dvmr, "flush_dvmr");
+    if (md->flush_dvmr != NULL) {
```

Comment:
is it still needed given the checks in line 1245-1246?

Reply:
done